src/main/java/com/sh/threesentences/users/service/AuthorityService.java

```diff
@@ -0,0 +1,43 @@
+package com.sh.threesentences.users.service;
+
+import static com.sh.threesentences.readingspace.exception.ReadingSpaceErrorCode.READING_SPACE_NOT_FOUND;
+import static com.sh.threesentences.topic.exception.TopicErrorCode.UNAUTHORIZED_TO_CREATE_TOPIC;
+
+import com.sh.threesentences.readingspace.entity.ReadingSpaceMemberRole;
+import com.sh.threesentences.readingspace.repository.UserReadingSpaceRepository;
+import com.sh.threesentences.users.entity.User;
+import com.sh.threesentences.users.exception.UserNotFoundException;
+import com.sh.threesentences.users.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class AuthorityService {
+
+    private final UserRepository userRepository;
+
+    private final UserReadingSpaceRepository userReadingSpaceRepository;
+
+    public ReadingSpaceMemberRole getReadingSpaceMemberRole(Long readingSpaceId, String email) {
+        User user = userRepository.findByEmail(email)
+            .orElseThrow(UserNotFoundException::new);
+        return userReadingSpaceRepository.findByUserAndReadingSpaceId(
+                user, readingSpaceId)
+            .orElseThrow(
+                () -> new IllegalStateException(READING_SPACE_NOT_FOUND.getMessage()));
+    }
```

Comment:
중요한 부분은 아니지만 줄바꿈이 굉장히 난해하게 들어가있네요.

```suggestion
        return userReadingSpaceRepository.findByUserAndReadingSpaceId(user, readingSpaceId)
            .orElseThrow(() -> new IllegalStateException(READING_SPACE_NOT_FOUND.getMessage()));
```

가독성도 좋지 않고 불필요하게 라인 수도 늘어나는것 같아요. editorconfig 때문인가요? 혹시 그렇다면 기본 설정을 좀 수정해보시면 좋겠습니다.